subprojects/android-test/test-app/build.gradle.kts

```diff
@@ -178,6 +178,33 @@ extensions.getByType<GradleInstrumentationPluginConfiguration>().apply {
             }
         }
     }
+
+    configurationsContainer.register("uiDebug") {
+        tryToReRunOnTargetBranch = false
+        reportSkippedTests = false
+        rerunFailedTests = false
+        reportFlakyTests = false
+        // uncomment after 2020.3.6 release (MBS-8050)
+        // debugMode = true
+
+        targetsContainer.register("api27") {
+            deviceName = "API27"
+
+            scheduling = SchedulingConfiguration().apply {
+                quota = QuotaConfiguration().apply {
+                    retryCount = 1
+                    minimumSuccessCount = 1
+                }
+
+                reservation = TestsBasedDevicesReservationConfiguration().apply {
+                    device = Emulator27
```

Comment:
We have the only one emulator for easier debugging.
It'd be great if we could pass a version through makefile.
Change build.gradle just to switch a version is inconvenient.

Reply:
I totally agree. We need to develop a convenient way to debug tests:
1. Run a specific test or the whole class
1. Choose API level

But I think the proposal is out of the scope of this task.

Reply:
Ok